tools/java/org/unicode/cldr/tool/SearchCLDR.java

```diff
@@ -204,6 +209,9 @@ public static void main(String[] args) {
             CLDRFile diffFile = null;
 
             if (checkCldr != null) {
+                if (locale.equals("eo")) {
```

Comment:
debug code… would be good to clean up in a future PR?  (By the way, you should be able to use conditional breakpoints to stop iff `locale.equals("eo")` ..)